`TestGraph::test_ts_fit_graph` failing with `xclim` v0.45.0

* RavenPy version: v0.12.2
* Python version: Any
* Operating System: All

### Description

Tracing the chain of events, it must have something to do with changes to `xclim.indicators.generic`'s `fit` and `stats`. 

### What I Did

```console
 _________________________ TestGraph.test_ts_fit_graph __________________________
[gw0] linux -- Python 3.11.5 /home/runner/work/RavenPy/RavenPy/.tox/py311-linux/bin/python
Traceback (most recent call last):
  File "/home/runner/work/RavenPy/RavenPy/.tox/py311-linux/lib/python3.11/site-packages/_pytest/runner.py", line 341, in from_call
    result: Optional[TResult] = func()
                                ^^^^^^
  File "/home/runner/work/RavenPy/RavenPy/.tox/py311-linux/lib/python3.11/site-packages/_pytest/runner.py", line 262, in <lambda>
    lambda: ihook(item=item, **kwds), when=when, reraise=reraise
            ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/RavenPy/RavenPy/.tox/py311-linux/lib/python3.11/site-packages/pluggy/_hooks.py", line 493, in __call__
    return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/RavenPy/RavenPy/.tox/py311-linux/lib/python3.11/site-packages/pluggy/_manager.py", line 115, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/RavenPy/RavenPy/.tox/py311-linux/lib/python3.11/site-packages/pluggy/_callers.py", line 152, in _multicall
    return outcome.get_result()
           ^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/RavenPy/RavenPy/.tox/py311-linux/lib/python3.11/site-packages/pluggy/_result.py", line 114, in get_result
    raise exc.with_traceback(exc.__traceback__)
  File "/home/runner/work/RavenPy/RavenPy/.tox/py311-linux/lib/python3.11/site-packages/pluggy/_callers.py", line 77, in _multicall
    res = hook_impl.function(*args)
          ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/RavenPy/RavenPy/.tox/py311-linux/lib/python3.11/site-packages/_pytest/runner.py", line 177, in pytest_runtest_call
    raise e
  File "/home/runner/work/RavenPy/RavenPy/.tox/py311-linux/lib/python3.11/site-packages/_pytest/runner.py", line 169, in pytest_runtest_call
    item.runtest()
  File "/home/runner/work/RavenPy/RavenPy/.tox/py311-linux/lib/python3.11/site-packages/_pytest/python.py", line 1792, in runtest
    self.ihook.pytest_pyfunc_call(pyfuncitem=self)
  File "/home/runner/work/RavenPy/RavenPy/.tox/py311-linux/lib/python3.11/site-packages/pluggy/_hooks.py", line 493, in __call__
    return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/RavenPy/RavenPy/.tox/py311-linux/lib/python3.11/site-packages/pluggy/_manager.py", line 115, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/RavenPy/RavenPy/.tox/py311-linux/lib/python3.11/site-packages/pluggy/_callers.py", line 113, in _multicall
    raise exception.with_traceback(exception.__traceback__)
  File "/home/runner/work/RavenPy/RavenPy/.tox/py311-linux/lib/python3.11/site-packages/pluggy/_callers.py", line 77, in _multicall
    res = hook_impl.function(*args)
          ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/RavenPy/RavenPy/.tox/py311-linux/lib/python3.11/site-packages/_pytest/python.py", line 194, in pytest_pyfunc_call
    result = testfunction(**testargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/RavenPy/RavenPy/tests/test_graphs.py", line 16, in test_ts_fit_graph
    np.testing.assert_array_equal(p.isnull(), False)
  File "/home/runner/work/RavenPy/RavenPy/.tox/py311-linux/lib/python3.11/site-packages/numpy/testing/_private/utils.py", line 985, in assert_array_equal
    assert_array_compare(operator.__eq__, x, y, err_msg=err_msg,
  File "/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/contextlib.py", line 81, in inner
    return func(*args, **kwds)
           ^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/RavenPy/RavenPy/.tox/py311-linux/lib/python3.11/site-packages/numpy/testing/_private/utils.py", line 862, in assert_array_compare
    raise AssertionError(msg)
AssertionError: 
Arrays are not equal

Mismatched elements: 2 / 2 (100%)
 x: array([[ True],
       [ True]])
 y: array(False)
```

--- a/HISTORY.rst
+++ b/HISTORY.rst
@@ -2,9 +2,12 @@
 History
 =======
 
+0.12.4 (unreleased)
+-------------------
+* In tests, set xclim' missing value option to ``skip``. As of xclim 0.45, missing value checks are applied to ``fit`` indicator, meaning that parameters will be set to None if missing values are found in the fitted time series. Wrap calls to ``fit`` with ``xclim.set_options(check_missing="skip")`` to reproduce the previous behavior of xclim.
+
 0.12.3 (2023-08-25)
 -------------------
-
 * `RavenPy` now uses `platformdirs` to write `raven_testing` to the user's cache directory. Dynamic paths are now used to cache data dependent on the user's operating system. Developers can now safely delete the `.raven_testing_data` folder in their home directory without affecting the functionality of `RavenPy`.
 * Updated `raven-hydro` to v0.2.4 to address CMake build issues.
 

--- a/ravenpy/utilities/graphs.py
+++ b/ravenpy/utilities/graphs.py
@@ -330,6 +330,8 @@ def ts_fit_graph(ts: xr.DataArray, params: xr.DataArray) -> matplotlib.pyplot.Fi
     dist = params.attrs["scipy_dist"]
 
     fig, axes = plt.subplots(n, figsize=(10, 6), squeeze=False)
+    if params.isnull().any():
+        raise ValueError("Null values in `params`.")
 
     for i in range(n):
         ax = axes.flat[i]

--- a/ravenpy/utilities/nb_graphs.py
+++ b/ravenpy/utilities/nb_graphs.py
@@ -133,6 +133,8 @@ def ts_fit_graph(ts: xr.DataArray, params: xr.DataArray) -> matplotlib.pyplot.Fi
 
     ts = ts.isel(nbasins=0)
     params = params.isel(nbasins=0)
+    if params.isnull().any():
+        raise ValueError("Null values in `params`.")
 
     # Using matplotlib's default binning strategy
     hist, bins, mh = plt.hist(ts, bins="auto", density=True)

--- a/tests/test_graphs.py
+++ b/tests/test_graphs.py
@@ -1,5 +1,6 @@
 import numpy as np
 import xarray as xr
+from xclim import set_options
 from xclim.indicators.generic import fit, stats
 
 from ravenpy.utilities import graphs
@@ -12,7 +13,8 @@ class TestGraph:
         )
         with xr.open_dataset(f) as ds:
             ts = stats(ds.q_sim, op="max", freq="M")
-            p = fit(ts)
+            with set_options(check_missing="skip"):
+                p = fit(ts)
             np.testing.assert_array_equal(p.isnull(), False)
 
             fig = graphs.ts_fit_graph(ts, p)

--- a/tests/test_nb_graphs.py
+++ b/tests/test_nb_graphs.py
@@ -1,5 +1,6 @@
 import pytest
 import xarray as xr
+from xclim import set_options
 
 
 class TestNBGraphs:
@@ -24,6 +25,6 @@ class TestNBGraphs:
 
         with xr.open_dataset(get_local_testdata(self.hydrographs)) as ds:
             ts = stats(ds.q_sim.load(), op="max", freq="M")
-
-        params = fit(ts, dist="gamma")
+        with set_options(check_missing="skip"):
+            params = fit(ts, dist="gamma")
         self.nbg.ts_fit_graph(ts, params)

